Extension/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # C/C++ for Visual Studio Code Changelog
 
 ## Version 1.12.1 (pre-release): August 15, 2022
-
 ## New Features
 * Add Doxygen comment generation via command, context menu, code action, or typing. [#5683](https://github.com/microsoft/vscode-cpptools/issues/5683)
+* Add auto-complete of Doxygen keywords in comments.
```

Comment:
Maybe add something like "after `@` or `\`"?

Reply:
Isn't that implicit, as all Doxygen keywords have to start with those?

Reply:
I don't know...if you think so.